sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/_tests_/e2e/student-scholastic-standings.institutions.controller.saveScholasticStanding.e2e-spec.ts

```diff
@@ -341,26 +489,277 @@ describe("StudentScholasticStandingsInstitutionsController(e2e)-saveScholasticSt
     ]);
   });
 
+  it("Should not create new scholastic standing Student did not complete program for a part-time application when date of change is not between study start date and end date.", async () => {
+    // Arrange
```

Comment:
Please add a todo to use actual form.io container later as the core logic is inside it.